_includes/footer.html

```diff
@@ -37,6 +37,7 @@
                                 <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
                                 <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
+                                <li><a href="https://plus.google.com/108920596016838318216" target="default"><i class="icon social_googleplus"></i></a></li>
```

Comment:
For all the Other Links, Target is `_self`. Your one, it 's `default`. Better to maintain every social link as same.